### PR TITLE
LPS-96061 No longer needed double backslash.

### PR DESCRIPTION
**Pull request by marianoalvarosaiz**

### CODE REVIEW

**Comment by vicnate5** [comment]
@shuyangzhou this is for an FP5 bug, can you review soon?

File: modules/third-party/org-apache-felix-fileinstall/patches/LPS-96061.patch [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
++    private static final Pattern ESCAPED_CLOSING_CURLY = Pattern.compile("\\\\+\\}");
+ 
+ 
+     /**
+@@ -407,7 +411,10 @@ public class InterpolationHelper {
+     private static String unescape(String val)
+     {
+         val = val.replaceAll("\\" + MARKER, "\\$");
+-        int escape = val.indexOf(ESCAPE_CHAR);
++        if (!val.matches(".*\\$\\\\*\\{.*\\}.*")) {
```
Comment (shuyangzhou) [inline modules/third-party/org-apache-felix-fileinstall/patches/LPS-96061.patch]: @marianoalvarosaiz can you pull this ".*\\$\\\\*\\{.*\\}.*" out as a Pattern, and resend, thanks!

**Comment by marianoalvarosaiz** [comment]
see https://github.com/shuyangzhou/liferay-portal/pull/7996

**Pull request closed without merging**